### PR TITLE
Bump Roslyn to 2.9.0-beta4-63001-02

### PR DESCRIPTION
**Pull request by KirillOsenkov**

### CODE REVIEW

File: main/src/core/MonoDevelop.Core/MonoDevelop.Core.csproj [hunk 1]
```diff
@@ -101,16 +101,16 @@
       <HintPath>..\..\..\packages\Humanizer.Core.2.2.0\lib\netstandard1.0\Humanizer.dll</HintPath>
     </Reference>
     <Reference Include="SQLitePCLRaw.batteries_e_sqlite3">
-      <HintPath>..\..\..\packages\SQLitePCLRaw.bundle_e_sqlite3.1.1.10\lib\net45\SQLitePCLRaw.batteries_e_sqlite3.dll</HintPath>
+      <HintPath>..\..\..\packages\SQLitePCLRaw.bundle_e_sqlite3.1.1.11\lib\net45\SQLitePCLRaw.batteries_e_sqlite3.dll</HintPath>
```
Comment (Therzok) [inline main/src/core/MonoDevelop.Core/MonoDevelop.Core.csproj]: YES!

Comment (decriptor) [reply main/src/core/MonoDevelop.Core/MonoDevelop.Core.csproj]: Is this the stuff needed for the perf improvements?

Comment (Therzok) [reply main/src/core/MonoDevelop.Core/MonoDevelop.Core.csproj]: One of it, yes. There's an allocation optimization on typesystem database loading that I worked on, which should improve time to intellisense for big solutions.

**Comment by KirillOsenkov** [comment]
Companion PR is https://github.com/xamarin/md-addins/pull/3232 - merge that first, then bump md-addins, then merge this PR.

**Pull request merged**